grakn-engine/src/main/java/ai/grakn/engine/session/RemoteSession.java

```diff
@@ -97,12 +97,11 @@ private void startSession(Json json) {
         if (sessionAuthorised(json)) {
             String keyspace = json.at(REST.RemoteShell.KEYSPACE).asString();
             String outputFormat = json.at(REST.RemoteShell.OUTPUT_FORMAT).asString();
-            boolean showImplicitTypes = json.at(REST.RemoteShell.IMPLICIT).asBoolean();
```

Comment:
Now you can kill it on the client-side as well (that's class `GraqlShell`), plus the docs.